README.rst

```diff
@@ -54,8 +54,15 @@ Next to open a terminal at the project root and install the requirements for loc
     $ pip install -r requirements/local.txt
 
 Next to create MySQL database::
+    # if you are using Ubuntu 14.04, you may need to find a workaround for the following two commands
+    $ sudo systemctl start mariadb
+    $ sudo systemctl enable mariadb
     
-    $ echo "CREATE DATABASE poradnia CHARACTER SET utf8 COLLATE utf8_polish_ci;" | mysql
+    $ mysql_tzinfo_to_sql /usr/share/zoneinfo | sudo mysql mysql
```

Comment:
Zdaje się, że tutaj akurat musi zostać tak jak jest. Pierwsze `mysql` to komenda, drugie to baza (która akurat nazywa się mysql).